docs/supplychain-sbom.md

```diff
@@ -448,27 +457,27 @@ Not to be confused with the [SBOM Author](#sbom-author--role-) role.
-| 🟥 | Component Name                 | Yes      | Text         | bom.components[].name                                                 |      | NTIA-SBOM, DE-TR.5.2.2, CRA-AV             |
-| 🟥 | Version                        | Yes      | Text         | bom.components[].version                                              |      | NTIA-SBOM, DE-TR.5.2.2                     |
-| 🟥 | Dependencies                   | Yes      | List         | bom.components[], bom.dependencies[]                                  |      | CRA-AII(5), NTIA-SBOM                      |
+| 🟥 | Component Name                 | Yes      | Text         | bom.components[].name                                                 | packages[].name | NTIA-SBOM, DE-TR.5.2.2, CRA-AV             |
+| 🟥 | Version                        | Yes      | Text         | bom.components[].version                                              | packages[].versionInfo | NTIA-SBOM, DE-TR.5.2.2                     |
+| 🟥 | Dependencies                   | Yes      | List         | bom.components[], bom.dependencies[]                                  | relationships[].[spdxElementId,relatedSpdxElement,relatedSpdxElement] | CRA-AII(5), NTIA-SBOM                      |
 | 🟥 | Security contact               | Yes      | URL          | bom.externalReferences[].security-contact                             |      | CRA-AII(2)                                 |
-| 🟥 | Unique ID, Product ID          | Yes      | PURL         | bom.components[].purl                                                 |      | CRA-AII(3), NTIA-SBOM, CRA-AV              |
-| 🟥 | Purpose, Intended Use          | Yes      | Text         | bom.components[].description                                          |      | CRA-AII(4)                                 |
-| 🟨 | Licenses                       | Yes      | SPDX License | bom.components[].licenses[]                                           |      |                                            |
-| 🟥 | Public Code Repository         | Yes      |              | bom.metadata.component.externalReferences[].vcs                       |      |                                            |
+| 🟥 | Unique ID, Product ID          | Yes      | PURL         | bom.components[].purl                                                 | packages[].externalRefs.referenceCategory = "PACKAGE-MANAGER", packages[].externalRefs.referenceType = "purl", packages[].externalRefs.referenceLocator | CRA-AII(3), NTIA-SBOM, CRA-AV              |
+| 🟥 | Purpose, Intended Use          | Yes      | Text         | bom.components[].description                                          | packages[].comment | CRA-AII(4)                                 |
+| 🟨 | Licenses                       | Yes      | SPDX License | bom.components[].licenses[]                                           | packages[].licenseConcluded, packages[].licenseConcluded |                                            |
```

Comment:
Is this a typo?

Reply:
No, these are the fields you need to add a PURL. It's not very pretty, nor is it documented well
https://spdx.github.io/spdx-spec/v2.3-RC1/external-repository-identifiers/#f35-purl

I'm hopeful SPDX 3.0 is easier

Reply:
Ah, I was referring to the last line `packages[].licenseConcluded, packages[].licenseConcluded`, which has a duplicate as it is now.

Reply:
Argh, I had to of those. I Just fixed them. It should be licenseConcluded and licenseDeclared